### ISSUE TITLE
Reset previous log and create new section when logging

### ISSUE THREAD

**Issue by paulomann** [wandb/client#1132]
`wandb --version && python --version && uname`

* Weights and Biases version: 0.9.1
* Python version: 3.7.3
* Operating System: Linux Ubuntu 16.04.5 LTS (Xenial Xerus)

### Description

Hi, I am new to this framework, but I can't find a place to answer my two questions, so here I am:

**1.** I would like to know if there is an option to overwrite the log. For example, in **Training 1** I run many `wandb.log({"loss": loss, "val_loss": val_loss}, step=epoch_i)`. Then, I want to run a **Training 2** experiment with different hyperparameters, can I overwrite the log under the same run name? I don't want to _append_, I want to _reset_ the **Training 1** log.

**2.** When using `wandb.log` function, can I already redirect some logs to specific sections to better visualize on the webpage?

**Comment by vanpelt** [comment]
Hey @paulomann, anytime you call `wandb.init` we create a new run in wandb.  Subsequent calls to wandb.log will be associated with that run.  Each section of a report has a Run Set associated with it.  These Run Sets can be filtered by tags, config, or any metrics passed to log.  So you could do something like the following to achieve what you want.

```
wandb.init(project="my_project", config={"experiment": "my_unique_experiment"}, tags=["section1"], reinit=True)
wandb.log({"metric": 1})
wandb.init(project="my_project", config={"experiment": "my_unique_experiment"}, tags=["section2"], reinit=True)
wandb.log({"metric": 1})
```

Then you could configure the RunSets in the report to include all runs with `config.experiment=my_unique_experiment`, and each section could further filter to only runs with the "section1" or "section2" tags.

**Comment by paulomann** [comment]
Thanks @vanpelt , it helped a lot!
